### DISCUSSION TITLE
Dishwasher always sends triggers on Door open/closed

### DISCUSSION THREAD

**Discussion by dinneatgithub** [thoukydides/homebridge-homeconnect#165] [Q&A] [answered]
Since the last update of my dishwasher firmware, which was automatically installed yesterday, I always receive notifications when the door is opened or closed. Before the update, there was just a notification if the dishwasher was running or in delayed start mode.

As I don't need the notification: can I disable the door open/close notification generally? I couldn't find a config option in the homebridge/plugins section.

**Comment by thoukydides** [comment, marked as answer]
The door open notifications are generated by the Apple Home app. If you don’t want them then you need to change the configuration within the Home app, not within the plugin configuration.

Go to the Home Settings (from the ... icon at the top of the screen), and then under Doors you will see a list of accessories, and can then toggle the Activity Notifications option. This is configured separately on each iPhone/iPad.

This has been discussed elsewhere, such as in issue #132.

**Comment by dinneatgithub** [comment]
Thank you so much @thoukydides! I didn't expect to configure it there. Switched off and now family is happy again :-)

**Comment by dinneatgithub** [comment]
topic can be closed
